README.md

```diff
@@ -149,6 +149,12 @@ Rallies can be stacked indefinitely, but as soon as a Lad performs a normal Upri
 A Lord cannot use Rallies even if they are performing an Uprising flip to try and finish the nail. Doing so does not reset the Rally count.
 
 
+### Resetting Nails
+
+A Lad may skip their turn to reset their nail to its starting depth and angle. They may also choose to put a new Lad nail in to the starting depth and remove their existing nail
+
+A Lord may skip their turn to reset a single center nail to its starting depth and angle. They may also choose to put a new center nail in to the starting depth and remove the center nail in question. A Lord can do this only once during each of their reigns. In other words, if this ability is used by a Lord, that Lord may not use the ability again unless they are demoted and become a Lord again
```

Comment:
Some alternative ideas to prevent frivolous resets:
- The nail must be obviously bent beyond repair
- A majority vote of lords must pass

Reply:
Or maybe the cost should be drink related? Like all lords must drink to reset a nail?

Reply:
I like the majority vote idea. During their turn a Lord can choose to either 1) toss/swing like normal or 2) initiate a vote among all the Lords to reset a center nail. If the vote passes the nail is reset and the Lords turn is over. If the vote fails the Lord drinks and their turn is over.

Reply:
I also like the vote idea rather than drinking as the cost. IMO drinking is barely a cost, and also I like the idea that the game can still function mechanically without the drinking rules.

That said, since it's a vote I wonder if skipping a Lord's turn for the vote is too steep. It doesn't feel too powerful to be able to reset the nail and still be able to go since it benefits everyone. I'm down to playtest it as starting a vote = skip turn to see if how it works out though

Reply:
Updated the wording for this